partner_compassion/models/res_partner.py

```diff
@@ -0,0 +1,30 @@
+##############################################################################
```

Comment:
This should be merged with file `partner_compassion` because it extends the same model!